specification/schemaregistry/SchemaRegistry/tspconfig.yaml

```diff
@@ -0,0 +1,39 @@
+parameters:  
+  "service-dir":
+    default: "sdk/schemaregistry"
+  "dependencies":
+    "additionalDirectories": []
+    default: ""
+emit:
+  - "@azure-tools/typespec-autorest"
+linter:
+  extends:
+    - "@azure-tools/typespec-azure-core/all"
+options:
+  "@azure-tools/typespec-autorest":
+    azure-resource-provider-folder: "data-plane"
+    emitter-output-dir: "{project-root}/../data-plane/SchemaRegistry/preview/2023-07-01"
+    output-file: "SchemaRegistry.json"
+    examples-directory: examples
+  "@azure-tools/typespec-python":
+    package-dir: "azure-schemaregistry"
+    package-name: "{package-dir}"
+    package-version: 1.3.0b4
+    emitter-output-dir: "{project-root}/{service-dir}/{package-name}"
```

Comment:
Is this the right output directory? I think you want it under a generated directory right?

Reply:
I just remembered you're patching this so disregard the original comment. But do you need to specify this property? I dont think you should have to. Having the package-dir should be enough

Reply:
I'll try generating with just the package-name specified and see if there's an issue.